.github/workflows/publish_openrpc_document.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish OpenRPC document
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build_release_binary_and_publish_document:
+    name: Build nimiq-rpc-schema with release flag
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install protoc
+        run: |
+          sudo apt-get update && sudo apt-get install -y protobuf-compiler
+      - uses: actions-rs/toolchain@v1
```

Comment:
This action is no longer maintained, could you switch to `dtolnay/rust-toolchain@stable`?